### PR TITLE
chore: clean up the frontend package

### PR DESCRIPTION
**Pull request by im-klau**
## Name PR according scheme: type of changes (feat/fix/refactor etc.) + description
chore: clean up the frontend package

## Jira issue

JIRA issue: [FSU-61](https://klau.atlassian.net/browse/FSU-61)

## Describe your changes

This PR simply removes all the files that we won't use in this project.
It also imports the `Badge` component into the `index.jsx` file (Home page in the future) just to make sure it works 🎉 

## Checklist before requesting a review

- [ ] Checked which branch you would merge.
- [ ] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [ ] Performed a self-review of code (bugs, clean code rules, etc).
- [ ] Checked if added variables have unique names.
- [ ] Checked if added code/function is not a duplicate. 
- [ ] Tests have been added if possible.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/frontend/app/pages/index.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossOrigin />
+        <title>{homepage.attributes.seo.metaTitle}</title>
+        <meta
+          name="description"
+          content={homepage.attributes.seo.metaDescription}
+        />
+      </Head>
+      <Normalize />
+      <h1>{homepage.attributes.hero.title}</h1>
+      <Badge text="Badge" />
```
Comment (paulgrym) [inline packages/frontend/app/pages/index.jsx:23]: Awesome! Our component finally in use 😄

**Review by paulgrym — APPROVED** [review]
Great job! Now it's almost as clean as after Christmas cleaning  🎄🤣🎄

File: packages/frontend/app/.eslintrc [hunk 2]
```diff
@@ -1,11 +1,6 @@
...
-    "next",
-    "prettier"
-  ],
-  "plugins": [
-    "prettier"
-  ],
+  "extends": ["next", "prettier"],
+  "plugins": ["prettier"],
   "rules": {
     "prettier/prettier": "error"
```
[the remark this answers is not in the document]
Comment (im-klau) [reply packages/frontend/app/.eslintrc]: Hmm I think this change wasn't intentional, I've just merged the `main` branch and it's already there 🚀 Can you check this out once again, pls? 🙏

[the remark this answers is not in the document]
Comment (paulgrym) [reply packages/frontend/app/.eslintrc]: Now it works fine for me :D

**Review by paulgrym — APPROVED** [review]

**Review by nata-kab — APPROVED** [review]
It seems that everything works fine.🙌

**Pull request merged**